tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -286,16 +234,20 @@ build_compressed_scan_pathkeys(SortInfo *sort_info, PlannerInfo *root, List *chu
 			nulls_first = false;
 		}
 
-		/* Prepend the ec class for the sequence number. We are prepending
-		 * the ec for efficiency in finding it. We are more likely to look for it
-		 * then other ec classes */
-		prepend_ec_for_seqnum(root, info, sort_info, var, sortop, nulls_first);
+		/*
+		 * Create the EquivalenceClass for the sequence number, so that we can
```

Comment:
I wonder if _sequence number_ is the right term here. Wouldn't _compressed chunk_ be more appropriate?

Reply:
I'll update it to say "the sequence number column". The rest of the pathkeys is fulfilled by sorting on this column, so here we create a EC for it.